src/components/common/LinkButton/style.module.scss

```diff
@@ -0,0 +1,62 @@
+@use '../../../styles/vars.scss' as vars;
+
+.button {
+  align-items: center;
+  background-color: #62b0ff;
+  border-radius: 8px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.25);
+  color: #fff;
+  cursor: pointer;
+  display: flex;
+  height: 2rem;
+  justify-content: center;
+  padding: 4px 1rem;
+  transition: 0.3s ease-in-out transform;
+
+  width: fit-content;
+
+  &[data-size='default'] {
+    height: 2.5rem;
+  }
+
+  &[data-size='small'] {
+    height: 2rem;
+  }
+
+  &[data-variant='primary'] {
+    background-color: #62b0ff;
+    border: 1px solid #fff;
+    color: #fff;
+
+    &[data-destructive='true'] {
+      background-color: #ef626c;
+      border: 1px solid #fff;
+      color: #fff;
+    }
+  }
+
+  &[data-variant='secondary'] {
+    background-color: #fff;
+    border: 1px solid #62b0ff;
+    color: #62b0ff;
+
+    &[data-destructive='true'] {
+      background-color: #fff;
+      border: 1px solid #ef626c;
+      color: #ef626c;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.04);
+
+    &:disabled {
+      cursor: wait;
+      transform: scale(1);
+    }
+  }
+
+  &:disabled {
+    background-color: vars.$disabled !important;
```

Comment:
seeking to learn: why does this need to be `!important`?

Reply:
because chaining two attribute selectors like `[data-variant='secondary'][data-destructive='true']` appears to have greater precedence than `:disabled`

A way to avoid this could be to set a CSS variable to the background color, then if the button isn't disabled, set the background color to the CSS variable